lib/fabric/app/models/fabric/charge.rb

```diff
@@ -12,7 +12,7 @@ class Charge
     belongs_to :payment_intent, class_name: 'Fabric::PaymentIntent',
       primary_key: :stripe_id
     belongs_to :balance_transaction, class_name: 'Fabric::BalanceTransaction',
-      primary_key: :stripe_id
+      primary_key: :stripe_id, inverse_of: nil
```

Comment:
You don't have to define `balance_transaction_id` because that is the default. You need to define `stripe_id` because on the BalanceTransaction it would default to looking at the `_id` field.